src/main/java/im/juniq/stringcalculator/Calculator.java

```diff
@@ -0,0 +1,80 @@
+package im.juniq.stringcalculator;
+
+public class Calculator {
+	private static final String SEPARATOR = " ";
+	private String input;
+
+	public Calculator(String input) {
+		this.input = input;
+	}
+
+	public int run() {
+		validInputIsNullAndBlank();
+		String[] terms = extractToTerms(input);
+		return calculateEachTerm(terms);
+	}
+
+	private void validInputIsNullAndBlank() {
+		if (input == null || "".equals(input)) {
+			throw new IllegalArgumentException();
+		}
+	}
+
+	private String[] extractToTerms(String input) {
+		return input.split(SEPARATOR);
+	}
+
+	private int calculateEachTerm(String[] terms) {
+		int criterion = Integer.valueOf(terms[0]);
+		for (int i = 1; i < terms.length - 1; i += 2) {
+			criterion = calculate(criterion, terms[i], Integer.valueOf(terms[i + 1]));
```

Comment:
```suggestion
		int criterion = Integer.parseInt(terms[0]);
		for (int i = 1; i < terms.length - 1; i += 2) {
			criterion = calculate(criterion, terms[i], Integer.parseInt(terms[i + 1]));
```

Integer.valueOf 메서드의 반환값은 Integer입니다. 
사용하는 타입이 원시타입(int)라면 Integer.parseInt 메서드를 사용해서 불필요한 캐스팅을 줄이는게 좋겠습니다. 😄